rbfe_tutorial/cli_tutorial.md

```diff
@@ -146,7 +152,7 @@ partial_charge:
 2. Plan your rbfe network with an additional `-s` flag for passing the settings:
 
 ```bash
-openfe plan-rbfe-network -M tyk2_ligands.sdf -p tyk2_protein.pdb -o network_setup -s settings.yaml
+openfe plan-rbfe-network -M tyk2_ligands.sdf -p tyk2_protein.pdb -o network_setup --n-protocol-repeats=1 -s settings.yaml
```

Comment:
In the output below, we are missing this line: https://github.com/OpenFreeEnergy/openfe/blob/e369a20b19e1d0e1cb6b40d0a89f79527c912c09/openfecli/commands/plan_rbfe_network.py#L232